examples/rbd/fs-pod-rox.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: apps/v1
```

Comment:
rename this file to start with `pod-fs`

Reply:
There is already block-pod-clone.yaml, Hence I mad up block-pod-rox.yaml and fs-pod-rox.yaml :-)